Pod/Classes/OTNoop.m

```diff
@@ -24,7 +24,9 @@ @implementation OTNoopSpan
 - (id<OTTracer>)tracer { return g_defaultNoopTracer; }
 - (id<OTSpanContext>)context { return g_defaultNoopSpanContext; }
 - (void)setOperationName:(NSString*)operationName {}
-- (void)setTag:(NSString*)key value:(NSString*)value {}
+- (void)setTag:(NSString*)key stringValue:(nonnull NSString *)value {}
```

Comment:
I don't spend much time in Objective C, but my impression was that this sort of multi-type support is typically provided via a `NSObject *` parameter... at least that's how NSDictionary and friends work, correct?

Are there examples of Objective C APIs that do this sort of strongly-typed overloading?